tests/unit/test_app.py

```
@@ -910,19 +903,18 @@ def test_cache_invalidation_flow(client, setup_tabs_and_feeds):
             client.get("/api/tabs")
-            mock_tab_query_after_invalidation.order_by.return_value.all.assert_called_once(
-            )
+            mock_tab_query_after_invalidation.order_by.return_value.all.assert_called_once()
 
 
 # --- Tests for Model Methods ---
 
 
 def test_feed_item_to_dict_serialization(
-        client, ):  # client fixture ensures app_context and db setup
+    client,
```

Ruff (0.15.2)

[warning] Unused function argument: `client`

(ARG001)